benchmark/src/main/scala/keycloak/scenario/CommonSimulation.scala

```diff
@@ -27,6 +27,10 @@ abstract class CommonSimulation extends Simulation {
     var default = http
       .acceptHeader("application/json")
       .disableFollowRedirect
+      .shareConnections
+      .connectionHeader("keep-alive")
+      .useAllLocalAddresses
+      .disableCaching
```

Comment:
more real world?

https://gatling.io/docs/gatling/reference/current/http/protocol/#disablecaching

Reply:
well, wouldn't the real world be actually something like browser caching 
```
Expires header
Cache-Control header
Last-Modified header
ETag
```

Reply:
Hi @ddurham2 - my take on these parameters: 

* shareConnections: This would share connections across multiple clients, that is: different users would use the same HTTP connection. While this allows more requests per load driver, and prevents problems with the TCP_WAIT state of connections, it delivers false results about the target as you won't generate the overhead of a TLS handshake. Therefore I advise against it. 
BTW: As long as you use local addresses (like connecting to a local minikube), the TCP_WAIT problem doesn't occur.

* connectionHeader("keep-alive"): This is actually optional in HTTP/1.1 and not needed to have keep alive working. I recently debugged it, and I don't see a need to transmit it while still getting the keep-alive behavior.

* useAllLocalAddresses: I would be OK to use it, if it was chosen to create more connections from a single host. If this is the case, I'd like to see a comment in the code explaining the reasoning behind it.

* disableCaching: I wouldn't use it, as real browsers would cache. At the same time I'd like to learn more why you've chosen it.

Reply:
thanks @ahus1 for sharing your view on these parameters, I am not very well versed with all these parameters except `disableCaching` and I agree with your reasoning of using some of it here.

Reply:
fair enough.

I've adjusted things and put sharedConnections behind a --shared-connections=true argument